content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -74,13 +77,89 @@ If you notice data is missing in your bucket, do the following:
 - Verify the timestamps match the [precision parameter](/influxdb/cloud/write-data/#timestamp-precision).
 - Minimize payload size and network errors by [optimizing writes](/influxdb/cloud/write-data/best-practices/optimize-writes/)
 
-### Troubleshoot rejected points
+### Review rejected points
+
+InfluxDB may have rejected points even if the HTTP request returned "Success".
+Your organization's `_monitoring` bucket logs rejected data points and any parsing errors associated with them.
+To get a log of rejected points, query for the [`rejected_points` measurement](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) and the [`count` field](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
+Each entry has a `count` equal to `1`.
+
+```js
+from(bucket: "_monitoring")
+  |> range(start: -1h)
+  |> filter(fn: (r) => r._measurement == "rejected_points")
+  |> filter(fn: (r) => r["_field"] == "count")
+```
+
+InfluxDB returns `rejected_points` entries that contain the `bucket` where the rejection occurred and the `reason` for the rejection.
+If a parsing error occurred, the entry will contain an `error` field with details.
+
+```text
+rejected_points,bucket=YOUR_BUCKET,field=YOUR_FIELD,gotType=Float, \
+  measurement=YOUR_MEASUREMENT,reason=type\ conflict\ in\ batch\ write, \
+  wantType=Integer count=1i 1627906197091972750
+```
```

Comment:
Rather than line protocol, I wonder if it'd be better to show the table that would get returned from a query:

```suggestion
| _time                | _measurement    | measurement      | reason                       | bucket      | field      | gotType | wantType | _field | _value |
| :------------------- | :-------------- | :--------------- | :--------------------------- | :---------- | :--------- | :------ | :------- | :----- | -----: |
| 2021-08-02T12:09:57Z | rejected_points | YOUR_MEASUREMENT | type conflict in batch write | YOUR_BUCKET | YOUR_FIELD | Float   | Integer  | count  |      1 |
```